<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_10">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTableWidget" name="table"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>外观设置</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QCheckBox" name="show_line">
              <property name="text">
               <string>行表头</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="show_row">
              <property name="text">
               <string>列表头</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="color_change">
              <property name="text">
               <string>颜色交替</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="change_everyone">
              <property name="text">
               <string>单元格可编辑</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>选择设置</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QRadioButton" name="choose_everyone">
              <property name="text">
               <string>选择单元格</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="choose_line">
              <property name="text">
               <string>选择行</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>姓名</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="name"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>性别</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="sex"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>年龄</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="age"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>籍贯</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="address"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="add">
              <property name="text">
               <string>添加</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="insert">
              <property name="text">
               <string>插入</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="change">
              <property name="text">
               <string>修改</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="del">
              <property name="text">
               <string>删除</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
